### DISCUSSION TITLE
Does dble support work with MySQL  8.0 ?

### DISCUSSION THREAD

**Discussion by dcy10000** [actiontech/dble#2373] [Q&A] [answered]
dble could communicate with MySQL 8.0  normally. You could send queries to dble as normal.
But some newest features in 8.0 are not supported yet. Feature support are list as follows.

### Supported/Fixed
1. MySQL8.0's default authentication plugin named 'caching_sha2_password'.（support from version 2.19.05.0)
2. MySQL8.0 no longer supports either implicit or explicit sorting for GROUP BY.(fixed from version 2.19.11.17)
3. sync tx_isolation with MySQL lead to the problem of MySQL 8.0. (fixed from version 2.19.11.1)

### Not Supported
1. Window Function
2. Json Function
3. WITH (Common Table Expressions)
4. todo

**Comment by yanhuqing666** [comment, marked as answer]
### Supported/Fixed

1. MySQL8.0's default authentication plugin named 'caching_sha2_password'.（support from version 2.19.05.0)
2. MySQL8.0 no longer supports either implicit or explicit sorting for GROUP BY.(fixed from version 2.19.11.17)
3. sync tx_isolation with MySQL lead to the problem of MySQL 8.0. (fixed from version 2.19.11.1)

### Not Supported

1. Window Function
2. JSON Function
3. WITH (Common Table Expressions)
4. todo

**Comment by Dinosauria** [comment]
The newest mysql8.0.x version of mysql that tested supported by dble is 8.0.25 （Tested dble version is 3.21.10.1）
